cockroachcloud/pci-dss.md

```diff
@@ -0,0 +1,138 @@
+  <td><ol start="12">
+        <li>Maintain a policy that addresses information security for all personnel.</li>
+  </td>
+</tr>
+</tbody>
+</tgroup>
+</table>
+
+{{ site.data.products.dedicated }} is certified by a PCI Qualified Security Assessor (QSA) to be compliant with [PCI DSS 3.2.1](https://listings.pcisecuritystandards.org/documents/SAQ_D_v3_Merchant.pdf), as long as you take the actions outlined in [Responsibilities of the customer](#responsibilities-of-the-customer).
+
+## Responsibilities of Cockroach Labs
+
+Cockroach Labs takes actions to ensure that our operating procedures and the deployment environment for {{ site.data.products.dedicated }} clusters meet or exceed the requirements of PCI DSS 3.2.1. Some of these actions include:
+
+- Enforcing comprehensive security policies and standards.
+- Providing periodic security training for all Cockroach Labs employees.
+- Hardening our operating environments and networks according to industry standards and recommended practices, to ensure that they are secure and resilient against vulnerabilities and attacks.
+- Encrypting cluster data and metadata at rest and in transit.
+- Regularly scanning our environment using vendors designated by PCI as [Approved Scanning Vendors (ASVs)](https://www.pcidssguide.com/what-is-a-pci-approved-scanning-vendor-asv/) to ensure our continued compliance with PCI DSS 3.2.1, and correcting issues as quickly as possible.
+- Regularly scanning our environment and software for known security vulnerabilities and applying updates and security patches in a timely manner.
+- Implementing [data loss prevention (DLP)](https://pcidss.com/listing-category/data-loss-protection-dlp).
+- Logging cluster actions and events, redacting sensitive information in audit logs, and retaining audit logs according to the [logging requirements of PCI DSS 3.2.1](https://listings.pcisecuritystandards.org/documents/Effective-Daily-Log-Monitoring-Guidance.pdf).
+
+A comprehensive list of all actions that Cockroach Labs takes to ensure compliance with PCI DSS 3.2.1 is beyond the scope of this document. For more information, contact your Cockroach Labs account team.
+
+Compliance is a shared responsibility. Be sure to read [Responsibilities of the customer](#responsibilities-of-the-customer) to ensure that your cluster is compliant.
+
+## Responsibilities of the customer
+
+To ensure that a {{ site.data.products.dedicated }} cluster complies with PCI DSS 3.2.1, you must take additional steps, such as implementing security recommendations and carefully choosing business partners and vendors and ensuring that cardholder data is protected throughout its journey into and out of your {{ site.data.products.dedicated }} clusters.
+
+Cockroach Labs cannot provide specific advice about what is required for compliance with PCI DSS or how to implement a specific requirement. The following points help to illustrate some steps that organizations might take to help to ensure compliance.
```

Comment:
@adbrenn - Do you think this para is valid now given we know specific features that customers must implement to comply with PCI?

Reply:
I think this is still valid, because we can't advise the customer about the design of their own client apps, what other vendors they are using as part of their solution, etc. It is still possible for them to make a mistake somewhere that defeats the PCI-DSS requirements, right?